integ/src/nodegroup_provider.rs

```diff
@@ -620,3 +613,26 @@ async fn cluster_iam_identity_mapping(
 
     Ok(())
 }
+
+fn eks_role_policy_document(region: &str) -> String {
+    let principle = match region.contains(CHINA_REGION_SYMBOL) {
```

Comment:
```suggestion
    let principle = match region.starts_with(CHINA_REGION_PREFIX) {
```

Rather than `contains("cn")`, it might be safer to do something like `starts_with("cn-")`.